### PR TITLE
Added support of stateless components

### PR DESCRIPTION
**Pull request by kirovboris**
/cc @AlexanderMoskovkin

### CODE REVIEW

File: test/react-plugin-test.js [hunk 1]
```diff
@@ -39,18 +39,26 @@ test('Should get DOM node by composite selector', async t => {
...
-    var stateless = await ReactSelector.with({ timeout: 100 })('StatelessComponent');
-
-    await t.expect(stateless).notOk();
+test('Should get DOM node for stateless component', async t => {
+    var stateless1 = await ReactSelector('Stateless1');
+    var stateless2 = await ReactSelector('Stateless2');
+    var stateless3 = await ReactSelector('Stateless3');
+
+    await t.expect(stateless1.textContent).ok('test');
+    await t.expect(stateless2).ok();
```
Comment (AlexanderMoskovkin) [inline test/react-plugin-test.js]: What would you like to check here? 
If you want to get a snapshot you should call it as an async function:
```js
var stateless2 = await ReactSelector('Stateless2');  -> var stateless2 = await ReactSelector('Stateless2')();
```

Or I guess it's better to check that the element exists:
```js
await t.expect(stateless2.exists).ok();
```

**Comment by kirovboris** [comment]
FPR

File: test/fixtures/common-tests.js [hunk 2]
```diff
@@ -36,18 +36,22 @@ test('Should get DOM node by composite selector', async t => {
...
+test('Should get DOM node for stateless component', async t => {
+    await t.expect(ReactSelector('Stateless1').textContent).ok('test');
+    await t.expect(ReactSelector('Stateless2').exists).ok();
+    await t.expect(ReactSelector('Stateless3').exists).ok();
 });
 
+test('Should get DOM node for pure component', async t => {
+    var pureComponent = await ReactSelector('PureComponent');
+
+    await t.expect(pureComponent).ok();
```
Comment (AlexanderMoskovkin) [inline test/fixtures/common-tests.js]:
```js
await t.expect(pureComponent.exists).ok();
```

**Comment by kirovboris** [comment]
FPR

**Pull request merged**